NEWS.md

```diff
@@ -5,6 +5,21 @@ Python 3.6 support dropped
 
   afew stopped supporting the older python version 3.6.
 
+Handle DMARC report with empty spf or dkim XML nodes
+
+DMARC Filter: allow to define subject regexp
+
+  Some DMARC report mail subject have a prefix before "Dmarc Report" in
```

Comment:
There's two times subject in here. I'd probably write "Some DMARC report mails have... in the subject and were not...

Reply:
:+1: